src/playbooks/checks/check.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Run preflight checks
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Run Subuid/Subgid check script
+      command: "{{ playbook_dir }}/../../roles/subuid_subgid_check/files/foreman-subuid-subgid-check"
```

Comment:
This doesn't work because the file is on the control node, but the command runs on the target host. You need to transfer it to the target machine.

It may be better to write an Ansible module that performs the check and rely on Ansible to transfer the file there rather than a shell script.

Reply:
IMHO no need to write a module. You can `command` a `grep` call with `ignore_errors: true` and then form a nice error message with `assert`.

But yes, I would too prefer it being in "pure" Ansible, over an external script.

Reply:
I’ve now removed the external shell script and implemented the check entirely in Ansible. Doing the same for the #207 one as well. Thanks for the suggestion!